pkg/privilege/privileges/privileges.go

```diff
@@ -930,9 +914,6 @@ func (p *UserPrivileges) ShowGrants(ctx context.Context, sctx sessionctx.Context
 		u = user.AuthUsername
 		h = user.AuthHostname
 	}
-	if err := p.Handle.ensureActiveUser(ctx, u); err != nil {
```

Comment:
Will `ShowGrants` returns `ErrNonexistingGrant` if an user is not loaded to memory after deleting this line?

Reply:
show grants for current user should never fail, because the current user must login to execute commands.
show grants for someone else, the caller ensures `ensureActiveUser()` is called before calling this `ShowGrants` function.